addons/web/static/tests/views/fields/datetime_field_tests.js

```diff
@@ -573,4 +573,36 @@ QUnit.module("Fields", (hooks) => {
 
         assert.strictEqual(getInput().value, "٠٢/٠٨/٢٠١٧ ١١:٤٥:٠٠");
     });
+
+    QUnit.test("list datetime with date widget test", async (assert) => {
```

Comment:
No test for widget="date". widget="date" + a field of type "datetime" won't work i think
(contrarily to what is written in your commit message)

Reply:
Hello,

Sorry for the mistake, I forgot to update the PR message. After my discussion with @aab-odoo , we will now provide an option (`show_time`) instead of a widget for the datetime field.

Reply:
If I understand well. If you have widget="date" + a field of type "datetime", the time will be shown by default. Is it what we want?

Reply:
or maybe the defaultValue for showTime should be false for widget="date"?

Reply:
Sorry for the misleading title, but yes, you are correct. Initially, our goal was to complete this task with the `date` widget compatible with the `datetime` field using the `showTime` prop internally. However, after discussing with Aaron, we decided to provide only the option since we needed to use the `showTime` prop anyway.